Makefile

```diff
@@ -11,7 +11,14 @@ build: install ## Webpack build the project
 	mkdir -p dist
 	./node_modules/.bin/rollup -c
 
-publish: build ## Publish current version of EventDrops
+publish: build ## Publish current version of EventDrops (need to set a VERSION=[major|minor|patch])
+	@if [ -z ${VERSION} ]; then \
+		echo "Error: please set VERSION envvar to one of the following value: major, minor, or patch.\n"; \
+		exit 1; \
+	fi
+
+	npm version ${VERSION}
+	git push --tags
```

Comment:
You could use scripts to work with version in order to automatically push the tags for example

https://docs.npmjs.com/cli/version

``` json
"scripts": {
  "preversion": "npm test",
  "version": "npm run build && git add -A dist",
  "postversion": "git push && git push --tags && rm -rf build/temp"
}
```

Reply:
We put all commands in the `Makefile`, hence these hooks are already implemented in it. Yet, :+1: for launching tests before publishing. PR updated. :)